web/datapipeline/clusters_projector.go

```diff
@@ -103,3 +110,229 @@ func getHanaSIDs(c *cluster.Cluster) []string {
 
 	return sids
 }
+
+// parseClusterDetails parses the cluster details depending on the cluster type
+func parseClusterDetails(c *cluster.Cluster) (json.RawMessage, error) {
+	switch detectClusterType(c) {
+	case models.ClusterTypeHANAScaleUp:
```

Comment:
Wouldn't be more clear to have `case models.ClusterTypeHANAScaleUp, models.ClusterTypeHANAScaleOut` instead of this `fallthrough`?
Example: https://www.golangprograms.com/golang-switch-case-statements.html